src/exceptions.h

```diff
@@ -0,0 +1,76 @@
+#ifndef EXCEPTIONS_INL_H_
+#define EXCEPTIONS_INL_H_
+
+#include "objects.h"
+
+#include <bjvm.h>
+
+typedef enum {
+  ARITH_DIV_ZERO
+} arithmetic_exception_reason;
+
+void bjvm_raise_exception_object(bjvm_thread *thread, bjvm_obj_header *obj);
+
+// Helper function to raise VM-generated exceptions
+int bjvm_raise_vm_exception(bjvm_thread *thread, slice exception_name, slice msg_modified_utf8);
+
+// Raise an ArithmeticException.
+static inline void raise_arithmetic_exception(bjvm_thread *thread, arithmetic_exception_reason reason) {
```

Comment:
This extra parameter isn't necessary bc there's no case where the VM raises an arithmetic exception with a different message. (There are functions in `Math` which do, but they construct the exception manually.)